### ISSUE TITLE
HOWTOS tracking issue

### ISSUE THREAD

**Issue by marcvanzee** [google/flax#2116]
This issue tracks which HOWTOs we would like to add.

## Process
- If you like to work on  HOWTO, please create a PR for it and mention this issue in the PR description, once it is merged we will check the box below.
- If you think we should add a HOWTO, please reply to this issue and we will add it to the list below.

## HOWTOs

- [ ] Data-parallel training (see #1982). @gmittal mentioned they would like to add this based on #1982.
- [ ] Best practices for dynamic length inputs.

**Comment by marcvanzee** [comment]
Thanks @lkhphuc, I think that is a great suggestion and I added it to the list.

**Comment by MrinalTyagi** [comment]
@marcvanzee Can you describe a bit more on the Dropout guide (similar to BatchNorm guide)?
